Sources/Plasma/FeatureLib/pfPatcher/plManifests.cpp

```diff
@@ -52,24 +52,46 @@ Mead, WA   99021
 #   define MANIFEST(in, ex) in
 #endif // PLASMA_EXTERNAL_RELEASE
 
+#if HS_BUILD_FOR_MACOS
+#   define PL_EXECUTABLE_SUFFIX ".app"
+#else
+#   define PL_EXECUTABLE_SUFFIX ".exe"
+#endif
```

Comment:
The `.app` suffix is technically true for all Apple platforms, not specifically macOS (this is a meaningless distinction for the moment, but might make life easier in the future). Can also add in a no-suffix case for non-Apple, non-Windows platforms

```suggestion
#if HS_BUILD_FOR_APPLE
#   define PL_EXECUTABLE_SUFFIX ".app"
#elif HS_BUILD_FOR_WIN32
#   define PL_EXECUTABLE_SUFFIX ".exe"
#else
#   define PL_EXECUTABLE_SUFFIX ""
#endif
```

Reply:
I can make this change - but my thinking here is also that the Mac platform is the only platform where we can (realistically) self patch at this moment. But changing to Apple wouldn't hurt anything.